buffalo/cmd/build.go

```diff
@@ -4,12 +4,14 @@ import (
 	"archive/zip"
 	"bytes"
 	"context"
+	"errors"
```

Comment:
this line is causing the tests to fail, there is already an errors package declared below.